modules/ROOT/pages/_partials/gs-adding-users-proc.adoc

```diff
@@ -55,26 +55,26 @@ a|
 Red Hat recommends that you define users of the solution you are developing in the *{customer-sso-name}*.
 For example, if you use {PRODUCT_SHORT} to develop an solution named `my-app`, define all users of `my-app` in the *{customer-sso-name}* instance only.
 
-Typically, you only use the *{customer-sso-name}* for end users, however you have the required privileges to use it for developer related activities.
+Typically, you only use the *{customer-sso-name}* for end users; however, you have the privileges required to use it for developer related activities.
 // tag::excludeDownstream[]
-For example, if a developer wanted to use http://microcks.github.io/[Microcks], that developer could create associated users in this instance.
+For example, if a developer wanted to use http://microcks.github.io/[Microcks],that developer could create associated users in this instance.
 // end::excludeDownstream[]
 ====
 
 
 [[managing-users]]
 = Managing Users in the {rhmi-sso-name}
 
-You manage users in the context of an Red Hat Single Sign-On realm.
+You manage users in the context of a Red Hat Single Sign-On realm.
 A realm manages a set of users, credentials, roles, and groups.
-A user belongs to a realm, and can log in to this realm.
+A user belongs to a realm and can log in to this realm.
 
-=== Accessing the Red Hat Single Sign-On administration console
+== Accessing the Red Hat Single Sign-On administration console
 
 To access the Red Hat Single Sign-On administration console:
 
-. Navigate to the *Red Hat Solution Explorer* URL that you received in the *Welcome* email.
-If asked for credentials, login as the `customer-admin` user.
+. Navigate to the *Red Hat Solution Explorer* URL that you received in the welcome email.
```

Comment:
hmmm.. i wonder how we can highlight this very specific email

Reply:
Would italics work? I thought bold was only for GUI instances. I'm not sure, but I think Red Hat Solution Explorer is also affected.

@finp 
```suggestion
Navigate to the _Red Hat Solution Explorer_ URL that you received in the _Welcome Email_.
```

Reply:
I think italic for Welcome email works, but not for RH Solx, make sense?